tools/ci/docker/linux/Dockerfile

```diff
@@ -97,6 +97,15 @@ RUN mkdir riscv64-unknown-elf-gcc && \
   curl -s -L "https://static.dev.sifive.com/dev-tools/riscv64-unknown-elf-gcc-8.3.0-2019.08.0-x86_64-linux-ubuntu14.tar.gz" \
   | tar -C riscv64-unknown-elf-gcc --strip-components 1 -xz
 
+###############################################################################
+# Build image for tool required by SPARC builds
+###############################################################################
+FROM nuttx-toolchain-base AS nuttx-toolchain-sparc
+# Download the SPARC GCC toolchain prebuilt by Gaisler
+RUN mkdir sparc-gaisler-elf-gcc && \
+  curl -L "https://www.gaisler.com/anonftp/bcc2/bin/bcc-2.1.0-gcc-linux64.tar.xz" \
+  | tar -C sparc-gaisler-elf-gcc --strip-components 1 -xvJf
```

Comment:
```suggestion
  | tar -C sparc-gaisler-elf-gcc --strip-components 1 -xJ
```

Reply:
Speaking of not needed, the manual says that `J` is not needed too.

Reply:
thanks,it indeed solved problem,but J must need